pyflex/utils.py

```diff
@@ -71,3 +72,40 @@ def find_local_extrema(data):
 
     return np.array(sorted(list(maxs.union(set(maxima)))), dtype="int32"), \
         np.array(sorted(list(mins.union(set(minima)))), dtype="int32")
+
+
+def get_surface_wave_arrivals(dist_in_deg, min_vel, max_vel, ncircles=1):
+    """
+    Calculate the arrival time of surface waves, based on the distance
+    and velocity range (min_vel, max_vel).
+    This function will calculate both minor-arc and major-arc surface
+    waves. It further calcualte the surface orbit multiple times
+    if you set the ncircles > 1.
+
+    Returns the list of surface wave arrivals in time order.
```

Comment:
Can you add a short parameter description/type here? especially for units on vel